File: src/ch2/cmptext.py
afile = "./fizzbuzz_py.txt"
bfile = "./fizzbuzz_rs.txt"

with open(afile, "r") as fp:
    astr = fp.read()
with open(bfile, "r") as fp:
    bstr = fp.read()

astr = astr.strip()
bstr = bstr.strip()

if astr == bstr:
    print("OK")
else:
    print("NG")

File: src/ch2/cmptext.rs
use std::fs;

fn main() {
    let afile = "./fizzbuzz_py.txt";
    let bfile = "./fizzbuzz_rs.txt";

    let astr = fs::read_to_string(afile).unwrap();
    let bstr = fs::read_to_string(bfile).unwrap();

    let astr = astr.trim();
    let bstr = bstr.trim();

    if astr == bstr {
        println!("OK")
    } else {
        println!("NG")
    }
}
